airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftDestinationAcceptanceTest.java

```diff
@@ -72,7 +72,7 @@ protected JsonNode getFailCheckConfig() {
   }
 
   @Override
-  protected TestDataComparator getTestDataComparator() {
+  public TestDataComparator getTestDataComparator() {
```

Comment:
this is probably unnecessary, earlier CDK removed the protected visibility, now fixed.